Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM moexmen/node:6 AS node
```

Comment:
Why `node:6` instead of `node:8`? Do we have something that is not backwards compatible?

Reply:
Not sure if everything is compatible with npm 5... Apparently something died when @fonglh updated something.

Anyway... one thing at a time =P

Reply:
I might have done it wrongly, you can try again.

Reply:
Also, there's no need to put `AS node`

Reply:
in another PR? 😜